generate-renode-scripts.py

```diff
@@ -210,6 +210,10 @@ def generate_cpu(time_provider):
     cpuType: "rv32ima"
     privilegeArchitecture: PrivilegeArchitecture.Priv1_10
 """
+        elif variant == in ["i", "im", "ima", "imac"]:
```

Comment:
is this correct `== in`? Shouldn't it be just `variant in [...]`?

Reply:
Of course just "in". Pushed the correct one now.